index.rst

```diff
@@ -73,15 +73,14 @@ Contents
     :caption: General documentation
 
     general-doc/architecture.rst
-    general-doc/decentralized-web.rst
+    general-doc/thyra.rst
```

Comment:
I don't think it's a good idea to name the section Thyra: it's not self explanatory so it's more confusing than anything else. Also the name will change soon!

Reply:
My need is the following: we have published a new way of installing a node, using thyra. We must have a dedicated section to explain what it is. Once the rebranding is ready, all these sections will be revisited, improved and probably move elsewhere too (based on the communication strategy). 

So yes, thyra is not self-explanatory atm but it's a temporary solution to be consistent with other pages already published.

Reply:
I would suggest that either you make a dedicated section for Thyra and leave a section for Decentralized web or that you move Thyra under decentralized web and refer to it in other pages.

We'll probably revamp the documentation at some point but it'll be much easier if it's already well organised.